plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -40,7 +40,7 @@ var ErrNoNodesAvailable = fmt.Errorf("no nodes available to schedule pods")
 
 // implementation of the error interface
 func (f *FitError) Error() string {
-	output := fmt.Sprintf("failed to find fit for pod: %v", f.Pod)
+	output := fmt.Sprintf("failed to find fit for pod, ")
```

Comment:
The event is in the namespace of the pod so that seems redundant. Line separator is fine. Also the pod is the event.involvedObject so name is know from there. 

Sent from my iPhone

> On May 20, 2015, at 7:59 PM, Daniel Smith <notifications@github.com> wrote:
> 
> In plugin/pkg/scheduler/generic_scheduler.go:
> 
> > @@ -40,7 +40,7 @@ var ErrNoNodesAvailable = fmt.Errorf("no nodes available to schedule pods")
> >  
> >  // implementation of the error interface
> >  func (f *FitError) Error() string {
> > -	output := fmt.Sprintf("failed to find fit for pod: %v", f.Pod)
> > +	output := fmt.Sprintf("failed to find fit for pod, ")
> Print pod's namespace & name, and add line separators?
> 
> —
> Reply to this email directly or view it on GitHub.
>